netci.groovy

```diff
@@ -273,8 +280,8 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                         Utilities.addGithubPushTrigger(job)
                         break
                     case 'arm64':
+                        Utilities.addGithubPushTrigger(job)
```

Comment:
No, I think this was a copy/paste mistake. Looks like I switched addGithubPushTrigger and addEmailPublisher between here and the next line. I'll fix.